modules/contracts/src.ts/services/ethReader.ts

```diff
@@ -497,6 +497,7 @@ export class EthereumChainReader implements IVectorChainReader {
       if (!gasPrice) {
         try {
           gasPrice = await provider.getGasPrice();
+          gasPrice = gasPrice.add(gasPrice.mul(30).div(100))
```

Comment:
Move this to the top to be a declared constant.

Reply:
By "this" I mean the `30`.

Reply:
Should set constant to `0.3`.